index.js

```diff
@@ -45,11 +52,31 @@ var ERROR_TYPES = {
  * This library stores no default configuration of its own. It relies on the application/service
  * it is embedded in to supply this information.
  *
- * @param config
+ * @param appUA an identifying user agent string for your app, compatible with user agent formatting
+ * as per https://tools.ietf.org/html/rfc7231#section-5.5.3 e.g. 'my-app', 'my-app/0.1',
+ * 'my-app/0.1 (Ubuntu/12.04; nodejs/0.10.13)`
+ * @param config object containing config
  * @constructor
  */
-var PersonaClient = function (config) {
+var PersonaClient = function (appUA, config) {
+    if (_.isObject(appUA)) {
```

Comment:
Better to invert this: `!_.isString` then you catch all bad data types, not just objects.